src/uv-common.c

```diff
@@ -33,6 +33,44 @@
 # include <net/if.h> /* if_nametoindex */
 #endif
 
+#define ALIGN_VALUE(v, a)  ( ((v) + (a) - 1) & ~((a) - 1) )
```

Comment:
Yes, it only works on powers of 2. The original function this is meant to replace (_aligned_malloc) also has this requirement.